cmd/create.go

```diff
@@ -127,6 +127,10 @@ A full list of component types that can be deployed is available using: 'odo com
 			checkError(err, fmt.Sprintf("failed to push component: %v", componentName))
 
 			fmt.Printf("Component '%s' was created.\n", componentName)
+		} else if len(catalogType) != 0 && catalogType == "template" {
```

Comment:
what if ` len(catalogType) != 0` and  `catalogType` is different from `"template"`?

Reply:
other catalog type is `imagestream` so other conditions will handle that